charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -203,8 +200,3 @@ alfresco-content-services:
     APP_CONFIG_OAUTH2_SILENT_LOGIN: "\"true\""
     APP_CONFIG_OAUTH2_REDIRECT_LOGIN: "/workspace/"
     APP_CONFIG_OAUTH2_REDIRECT_LOGOUT: "/workspace/logout"
-
-alfresco-sync-service:
```

Comment:
We'll still need this for now.

Reply:
Sync is included in the ACS chart at this very moment.